### ISSUE TITLE
Documentation on website is hard to follow

### ISSUE THREAD

**Issue by tonyganchev** [bazelbuild/rules_pkg#820]
Consider the screenshot below. Text in the tables is wrapped aggressively and very hard to read. Can you switch to a var-width layout or come up with alternative doc rendering?
![image](https://github.com/bazelbuild/rules_pkg/assets/12876245/4d412208-9c47-4b98-8b86-65e996b54753)

**Comment by tonyaiuto** [comment]
I'm still hoping stardoc comes up with a new rendering tool.

**Comment by dieortin** [comment]
Having a sidebar with links to the different rules would also be great, I currently have to jump to the top and back down all the time
